### DISCUSSION TITLE
Camera Area

### DISCUSSION THREAD

**Discussion by TeacherRabbit123** [netless-io/flat#1844] [General]
One of the issues of teachers is that the camera are seems to be crammed. is it possible to make it look like this? Thanks! 
![camera](https://user-images.githubusercontent.com/126366852/221456392-4faee0b2-1251-444b-a7e8-680cff2f02e3.png)

**Comment by Leooeloel** [comment]
It will be supported in the next version, we are working on this feature.

**Reply by TeacherRabbit123** [reply]
Thanks, sir
